### PR TITLE
Feature/rework api

### PR DESCRIPTION
**Pull request by kristofsajdak**

### CODE REVIEW

File: src/vouch/api/crypto.clj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 (ns vouch.api.crypto
```
Comment (kristofsajdak) [inline src/vouch/api/crypto.clj]: Ideally the vouch-fabric-api becomes a more focused library ( which we can share with the public ), only concerning itself with blockchain interaction : transact / query and some helper functions. This involves changing the name of the lib and the namespaces, and factoring out the integration tests to a separate module )

File: test/vouch/api/vzp_test.clj [hunk 2]
```diff
@@ -0,0 +1,85 @@
+(ns vouch.api.vzp-test
```
Comment (kristofsajdak) [inline test/vouch/api/vzp_test.clj]: see earlier comment, can disappear over time

Comment (raymcdermott) [reply test/vouch/api/vzp_test.clj]: See https://github.com/ledgrinc/vzp-idp/issues/75

File: src/vouch/api/crypto.clj (hunk 1 above)
Comment (raymcdermott) [reply src/vouch/api/crypto.clj]: Ticket to motivate and track the change https://github.com/ledgrinc/vzp-idp/issues/75

**Pull request merged**